sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -325,6 +326,23 @@ def test_binomial():
 
     assert binomial(a, b).is_nonnegative is True
 
+    # issue #13980 and #13981
+    assert binomial(-7, -5) == 0
+    assert binomial(-23, -12) == 0
+    assert binomial(13/2, -10) == 0
+
+    assert binomial(-49, -51) == 0
```

Comment:
Not sure for this case whether it should be `0`, though @jksuom cleared above why we should prefer it to be `0`, but [wolfram apha](http://www.wolframalpha.com/input/?i=binomial(-49,+-51)) gives the answer as `1225`